src/cli.rs

```diff
@@ -75,6 +78,10 @@ impl Options {
                 .multiple(true)
                 .conflicts_with("q")
                 .help("Increases the level of verbosity (the max level is -vvv)"))
+            .arg(Arg::with_name("chdir")
+                 .short("c")
```

Comment:
I would like to try and match other terminal emulators with this flag. The standard seems to be `--working-directory`.